BookStoreAPI/Models/BookDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace BookStoreAPI.Models;
+
+public class BookDto
```

Comment:
DTO to nie jest część modelu, powinno to być w osobnym folderze. Dodatkowo na DTO powinna być zrobiona walidacja przy pomocy fluent validation lub atrubutów data annotations.